lib/CMakeLists.txt

```diff
@@ -93,12 +93,28 @@ execute_process(
     OUTPUT_VARIABLE arch_flag_lines
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 
+try_compile(
+    HAVE_RVV_INTRINSICS
+    ${CMAKE_BINARY_DIR}
+    ${CMAKE_SOURCE_DIR}/cmake/Checks/check-rvv-intrinsics.c
+)
+if(HAVE_RVV_INTRINSICS)
+    message(STATUS "Checking RVV intrinsics - found")
+else()
+    message(STATUS "Checking RVV intrinsics - not found")
+endif()
+
```

Comment:
I assume CMake doesn't have a function for this check yet. Would it make sense to only check for RVV if `"${arch_prefix}" STREQUAL "rv"`

Reply:
That would require moving the check into the loop and checking it for every rvv architecture, that seems a bit redundant and IMO it's easier to read when it's not crammed into the nested statements.

Reply:
I think we can leave it as is. If anyone runs into issues because CMake is way to slow, we can still fix it.